srcpkgs/giph/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'giph'
+pkgname=giph
+version=1.1.1
+revision=1
+build_style=gnu-makefile
+depends="ffmpeg xdotool slop libnotify procps-ng"
+short_desc="Simple video (gif, webm, mp4) recorder"
+maintainer="chtc <notnotcha0t1c@protonmail.com>"
+license="MIT"
+homepage="https://github.com/phisch/giph"
+distfiles="${homepage}/archive/refs/tags/${version}.tar.gz>${pkgname}-${version}.tar.gz"
```

Comment:
huh? What's with the trailing `>${pkgname}-${version}.tar.gz` ?

Please go through the [manual](https://github.com/void-linux/void-packages/blob/master/manual.md) once.

Reply:
It is used to specify the filename. In this scenario it's actually useless because the filename isn't different but I was copy and pasting from another template I made.